coralproject/talk
# Added defaultResolveType

## What does this PR do?

When a plugin is removed after it creates data, we don't want to error out, this plugin introduces some simple concepts (Defaults) that are used when the type is undefined.

## How do I test this PR?

- Run with respect plugin
- Respect some posts ;)
- Run without respect plugin
- Notice that the page still loads!
